Neos.ContentRepository.Core/Classes/Feature/NodeMove/Dto/RelationDistributionStrategy.php

```diff
@@ -41,6 +41,19 @@ public static function fromString(?string $serialization): self
             : self::STRATEGY_GATHER_ALL;
     }
 
+    public static function fromName(string $name): self
+    {
+        if (str_starts_with($name, 'RelationDistributionStrategy::')) {
+            $name = substr($name, strpos($name, '::') + 2);
+            foreach (self::cases() as $status) {
+                if ($name === $status->name) {
+                    return $status;
+                }
+            }
+        }
+        return self::STRATEGY_GATHER_ALL;
```

Comment:
Sorry, one more addition:
If we decide to keep the value based serialization we should at least fail if referring to an invalid value, rather than falling back to some strategy